.azure/pipelines/helix-matrix.yml

```diff
@@ -0,0 +1,61 @@
+# We only want to run full helix matrix on master
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
```

Comment:
Suggest instead
``` yaml
pr: none
trigger: none
schedules:
- cron: "0 */12 * * *"
  branches:
    include:
    - master
  always: true
```
Or perhaps two cron entries for specific times of the day (in UTC). Not sure whether this should run unconditionally (`always: true`) or only when new commits are in 'master'.

Reply:
We don't want to just have it run every time there's a commit to master?

Reply:
I thought the decision was to run this a couple of times a day, not on every commit.

Reply:
Could just run it once a day then, is there a reason to run it a couple times a day?

Reply:
I believe @Pilchie suggested twice / day to keep the build count reasonable while catching problems merges create relatively quickly